src/deployers/PlatformPluginsDeployer.cpp

```diff
@@ -1,24 +1,50 @@
 // library headers
 #include <linuxdeploy/core/log.h>
-#include <boost/filesystem.hpp>
 
 // local headers
 #include "PlatformPluginsDeployer.h"
+#include "linuxdeploy/util/util.h"
 
 using namespace linuxdeploy::plugin::qt;
 using namespace linuxdeploy::core::log;
 
 namespace bf = boost::filesystem;
 
+PlatformPluginsDeployer::PlatformPluginsDeployer(std::string moduleName,
+                                                 core::appdir::AppDir& appDir,
+                                                 bf::path qtPluginsPath,
+                                                 bf::path qtLibexecsPath,
+                                                 bf::path installLibsPath,
+                                                 bf::path qtTranslationsPath,
+                                                 bf::path qtDataPath):
+                                                 BasicPluginsDeployer(std::move(moduleName),
+                                                                      appDir,
+                                                                      std::move(qtPluginsPath),
+                                                                      std::move(qtLibexecsPath),
+                                                                      std::move(installLibsPath),
+                                                                      std::move(qtTranslationsPath),
+                                                                      std::move(qtDataPath)) {
+    // check if the platform plugins are set in env
+    const auto* const platformPluginsFromEnvData = getenv("PLATFORM_PLUGINS");
```

Comment:
As said above, I'd rather enforce the deployment of `libqxcb.so`. Instead of `PLATFORM_PLUGINS`, I would like to have some `EXTRA_PLATFORM_PLUGINS`.